src/base/memory_alloc.h

```diff
@@ -45,6 +45,7 @@
 #include <memory>
 #include <functional>
 #endif
+#include "memory_cwrapper.h"
```

Comment:
How about leave comment why you include `_cwrapper` iso `_wrapper` with a caution 'Don't change it to _wrapper'?

Reply:
I add a caution comment.